lib/fork.js

```diff
@@ -1,54 +1,43 @@
 'use strict';
 var childProcess = require('child_process');
 var Promise = require('bluebird');
-var path = require('path');
+var dirname = require('path').dirname;
+var join = require('path').join;
```

Comment:
yeah, did that before, but we need `path` here - https://github.com/sindresorhus/ava/pull/236/files#diff-a48173105195b7a6481a3ad85b45144eR13

When we will support #229, `babel` variable name won't be a good choice.